scripts/zygote.ps1

```diff
@@ -35,6 +35,9 @@ function help() {
     Write-Host " ➡️ new"
     Write-Host "`t Create a new project from a template using TUI"
```

Comment:
Here should be TUI or GUI?

Reply:
yeah, should be TUI because is a Terminal Graphical Interface, is not Graphical at end